jsk_perception/CMakeLists.txt

```diff
@@ -18,6 +18,7 @@ find_package(catkin REQUIRED COMPONENTS
   angles
   cmake_modules
   cv_bridge
+  darknet
```

Comment:
@k-okada Updated image conversion from rgb8 uchar to float32 referring to your codes and it looks that it works well now. Thanks!